src/libraries/sendtohelix-browser.targets

```diff
@@ -239,6 +239,8 @@
         <Timeout Condition="'%(FileName)' == 'System.Collections.Immutable.Tests'">01:20:00</Timeout>
         <Timeout Condition="'%(FileName)' == 'System.Net.WebSockets.Client.Tests'">01:20:00</Timeout>
         <Timeout Condition="'%(FileName)' == 'System.Net.Http.Functional.Tests'">01:20:00</Timeout>
+        <Timeout Condition="'%(FileName)' == 'System.Numerics.Tensors.Tests'">01:20:00</Timeout>
```

Comment:
Why do we have the timeout set in both csproj and here?

Reply:
one is for helix and the other for xharness, I think